graphrag/index/graph/extractors/summarize/description_summary_extractor.py

```diff
@@ -127,15 +128,14 @@ async def _summarize_descriptions_with_llm(
     ):
         """Summarize descriptions using the LLM."""
         response = await self._llm(
-            self._summarization_prompt,
-            name="summarize",
-            variables={
+            self._summarization_prompt.format(**{
                 self._entity_name_key: json.dumps(items),
                 self._input_descriptions_key: json.dumps(
                     sorted(descriptions), ensure_ascii=False
                 ),
-            },
+            }),
+            name="summarize",
             model_parameters={"max_tokens": self._max_summary_length},
         )
         # Calculate result
-        return str(response.output)
+        return str(response.output.content)
```

Comment:
again, just curious, what is the type of content?

Reply:
str